### PR TITLE
Improve user content list styles

### PR DESCRIPTION
**Pull request by charrondev**
After discussing with @ElisaAlmeida I've updated the bullet list styles to be more inline with our mockups.

I also reduced the left margin from 3em to 2em to align with the mockups.

|Before|After|
|---|---|
|image|image|

### CODE REVIEW

File: library/src/scss/userContent/_lists.scss [hunk 1]
```diff
@@ -47,7 +47,14 @@ $nestedList-margin: 0 0 0 1em;
 .userContent {
     ul {
         li {
-            list-style: disc !important;
+            list-style: none !important;
+            position: "relative";
+
+            &::before {
+                content: `"•"`,
```
Comment (slafleche) [inline library/src/scss/userContent/_lists.scss]: Make sure to specify the font to something safe, like "arial" so it doesn't mess up if a bad font is selected.

File: applications/dashboard/design/style.css [hunk 2]
```diff
@@ -6214,15 +6198,9 @@ img.jcrop-preview {
 }
 
 .flex {
-  display: -webkit-flex;
-  display: -ms-flexbox;
   display: flex;
-  -webkit-justify-content: space-between;
```
Comment (slafleche) [inline applications/dashboard/design/style.css]: Looks like we lost a lot of auto prefixing? Did our settings change?

File: library/src/scss/userContent/_lists.scss (hunk 1 above)
Comment (charrondev) [reply library/src/scss/userContent/_lists.scss]: Done

File: applications/dashboard/design/style.css (hunk 2 above)
Comment (charrondev) [reply applications/dashboard/design/style.css]: Whoops. Fixed.

**Pull request merged**